modules/logging-release-notes-6-0-0.adoc

```diff
@@ -0,0 +1,121 @@
+// module included in logging-6-0-release-notes.adoc
+:_mod-docs-content-type: REFERENCE
+[id="logging-release-notes-6-0-0_{context}"]
+= Logging 6.0.0
+
+This release includes link:https://access.redhat.com/errata/RHBA-2024:6693[OpenShift Logging Bug Fix Release 6.0.0]
+
+[id="logging-release-notes-6-0-0-compatibility-support-matrix"]
+= Compatibility and support matrix
+
+In the table, components are marked with the following statuses:
+
+[horizontal]
+TP:: Technology Preview
+GA:: General Availability
+
+The components or features in link:https://access.redhat.com/support/offerings/techpreview[Technology Preview] status are experimental and are not intended for production use.
+
+[NOTE]
+====
+To know about compatible {product-title} versions for {logging-uc} releases, see link:https://access.redhat.com/product-life-cycles?product=Red%20Hat%20OpenShift%20Logging[Product Life Cycles].
+====
+
+.Components support matrix
+[options="header"]
+|===
+
+| {logging-uc} Version 6+| Component Version
+
+| Operator | `eventrouter` | `logfilemetricexplorer` | `loki` | `lokistack-gateway` | `opa-openshift` | `vector`
+
+|6.0 | 0.4 (GA) | v1.1 (GA) | v3.1.0 (GA) | v0.1 (GA) | v0.1 (GA) | v0.37.x (GA)
+
+|===
+
+[id="logging-release-notes-6-0-0-removal-notice"]
+== Removal notice
+
+In Logging 6.0, the support is removed for:
+
+* Red Hat managed Elasticsearch log storage solution. For example, elasticsearch-operator
+* Red Hat managed log visualization using Kibana and Console Log View Plugin
+* Fluentd-based log collector implementations
++
+[NOTE]
+====
+The log collector deployments based on Fluentd will migrate to Vector
+====
+* ClusterLogging.logging.openshift.io custom resource
+* ClusterLogForwarder.logging.openshift.io custom resource
+
+[NOTE]
+====
+In order to continue to use Elasticsearch and Kibana managed by the elasticsearch-operator, the administrator must modify those object's ownerRefs before deleting the ClusterLogging resource.
+====
+
+[id="logging-release-notes-6-0-0-enhancements"]
+== New features and enhancements
+=== Log Collection
+
```

Comment:
I feel like we should have a bullet point mentioning the introduction of the new CR since that's the major "new" feature of this release

Reply:
@JoaoBraveCoding Kindly point me out the the jira with updated release notes text and I will add it in the beginning.

Reply:
I don't see where we are documenting the epic https://issues.redhat.com/browse/LOG-3493  which implies API and the the majority of what drives this release.  Additionally there is no mention of https://issues.redhat.com/browse/LOG-4225  which is another epic witha new feature

Reply:
@jcantrill I have added https://issues.redhat.com/browse/LOG-3493. Kindly check the bottom of release notes that states tech Preview features separately, you will find https://issues.redhat.com/browse/LOG-4225.